acos_client/v30/slb/template/persistence.py

```diff
@@ -34,10 +34,10 @@ def exists(self, name):
         except acos_errors.NotFound:
             return False
 
-    def create(self, name, **kwargs):
+    def create(self, name, domain=None, **kwargs):
         if self.exists(name):
             raise acos_errors.Exists
-        self._post(self.prefix, self.get_params(name), **kwargs)
+        self._post(self.prefix, self.get_params(name, domain), **kwargs)
```

Comment:
This needs to expand the dict is passes, else the name param is going to be passed a dict.